bundle/internal/schema/annotations_openapi.yml

```diff
@@ -19,6 +19,9 @@ github.com/databricks/cli/bundle/config/resources.App:
   "description":
     "description": |-
       The description of the app.
+  "id":
```

Comment:
App now returns `id` as a unique identifier, we need to start using it in DABs

Reply:
@andrewnester How is the ID used elsewhere? AFAIK the name has been the "id" up to now.

Reply:
@pietern yes, the name is the identifier, this comment is mostly a reminder for myself to go check how it works and and if we should start using it